src/services/getFilmId.js

```diff
@@ -0,0 +1,3 @@
+export function getFilmId() {
```

Comment:
@Mmarvinn I would be put this function to `utils`, it is too small for `services`

Reply:
fix